interop/src/main.rs

```diff
@@ -17,17 +17,22 @@ mod clients;
 #[cfg(not(target_family = "wasm"))]
 mod util;
 
-#[cfg(not(target_family = "wasm"))]
-const TEST_SERVER_PORT: &str = "8000";
-#[cfg(not(target_family = "wasm"))]
-const TEST_SERVER_URI: &str = const_format::concatcp!("http://localhost:", TEST_SERVER_PORT);
-
 const MLS_MAIN_CLIENTID: &[u8] = b"test_main";
 const MLS_CONVERSATION_ID: &[u8] = b"test_conversation";
 const ROUNDTRIP_MSG_AMOUNT: usize = 100;
 
 const CIPHERSUITE_IN_USE: CiphersuiteName = CiphersuiteName::MLS_128_DHKEMX25519_AES128GCM_SHA256_Ed25519;
 
+#[cfg(not(target_family = "wasm"))]
+fn get_port() -> u16 {
+    8000 + uzers::get_current_uid() as u16
+}
+
+#[cfg(not(target_family = "wasm"))]
+fn get_test_server_uri() -> String {
+    format!("http://localhost:{}", get_port())
+}
+
```

Comment:
I'm not a fan of pulling in a new dependency just for this.
Why do we need to know the server port in advance?

What we should do is let `warp` bind to any free port it wants. If we really need that port somewhere later, we can get it from `warp` (see `bind_ephemeral`, which returns the bound address).